vyper/venom/passes/sccp/eval.py

```diff
@@ -0,0 +1,129 @@
+import operator
+from typing import Callable
+
+from vyper.utils import SizeLimits, evm_div, evm_mod
+from vyper.venom.basicblock import IROperand
+
+
+def _unsigned_to_signed(value: int) -> int:
```

Comment:
i think an assertion that the value is in bounds, otherwise we might let oob things in

Reply:
actually i would prefer use of `signed_to_unsigned` and `unsigned_to_signed` from `vyper.utils`